### PR TITLE
doc improvement: update current known issues for 1.7.0 release

### PR DESCRIPTION
**Pull request by greg-fer**
Added 1.7.0 label to the existing known issues.

Based on #4840 .

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

----

For known issues discovered during RC1 testing or before release, *open a PR that points to this branch*.

- [x] nRF9160 issues for Asset tracker checked by @nordic-hani and @jhn-nordic 
- [x] nRF9160 issues (Other) checked by @nordic-hani 
- [x] nRF5340 issue checked by @czeslawmakarski (related to 802.15.4)
- [x] Thread issues checked by @MarekPorwisz 
- [x] Zigbee issues checked by @maciejpietras 
- [x] Matter issues checked by @Damian-Nordic 
- [x] nRF Desktop and Pelion issues checked by @emob-nordic and @MarekPieta 
- [x] Bluetooth issue checked by @grochu 
- [x] BT mesh issues checked by @Balaklaka (another PR)
- [x] Build system issues checked by @hakonfam 
- [x] DFU and FOTA issues checked by @hakonfam 
- [x] SPM issues checked by @hakonfam 
- [x] MPSL issues checked by @rugeGerritsen 
- [x] Radio driver issues checked by @czeslawmakarski 
- [x] SD Controller issues checked by @rugeGerritsen 
- [x] SES issues checked by @tejlmand 
- [x] Toolchain issues checked by @tejlmand 

----

- [x] Edits from @umapraseeda from Sept 10, 11:11 (Teams)
- [x] Include issue for https://projecttools.nordicsemi.no/jira/browse/NCSDK-11234 requested by @MarekPieta 
- [x] Include #5598 from @hakonfam 
- [x] Add update that @jhirsi will provide on Monday, Sept 20, related to https://projecttools.nordicsemi.no/jira/browse/NCSDK-11033
- [x] Check if we need an entry for the installation on Linux - to be coordinated with @melwee01 after update from @mbolivar-nordic (checked with @carlescufi and @shantha-14 )

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -489,15 +490,15 @@ KRKNWK-6073: Potential delay during FOTA
...
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-10589: Android CHIPTool crashes when commissioning a Matter device
   In random circumstances, Android CHIPTool crashes when trying to connect to a Matter device over Bluetooth® LE.
 
   **Workaround** Restart the application and try to commission the Matter device again.
   If the problem persists, clear the application data and try again.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (Damian-Nordic) [inline doc/nrf/known_issues.rst:501]: It is no longer an issue in 1.7, but I'm not sure if we should update it here or in a separate PR (?)

**Review by Damian-Nordic — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst:501]: It's enough not to add the tag in this PR.

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -1132,7 +1133,7 @@ DRGN-15547: Assertion when updating PHY and the event length is configured too l
     * Set :c:union:`sdc_cfg_t` with :c:member:`event_length` to at least 2500 us if the application is using 1M PHY.
     * Set :c:union:`sdc_cfg_t` with :c:member:`event_length` to at least 7500 us if the application is using Coded PHY.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
Comment (rugeGerritsen) [inline doc/nrf/known_issues.rst:1136]: "DRGN-13594: The channel map provided by the LE Host Set Channel Classification HCI... " is fixed for NCS 1.7
```suggestion
.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```

**Comment by rugeGerritsen** [comment]
More known issues to be added under the section "SoftDevice Controller" which are fixed for NCS 1.7:
```
.. rst-class:: v1-6-1 v1-6-0

DRGN-15993: Assertion when using legacy advertising commands after HCI LE Clear Advertising Sets.

  **Workaround:** Don't mix legacy and extended advertising HCI commands.

.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0

DRGN-16079: Assertion when in LLPM mode and the connection interval was more than 1 ms.

  **Workaround:** Only use 1 ms connection interval when in LLPM mode.
```

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -338,22 +339,22 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (maciejpietras) [inline doc/nrf/known_issues.rst:342]: KRKNWK-8211 shall be removed from NCS1.7.0

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -338,22 +339,22 @@ KRKNWK-6408: ``diag`` command not supported
...
   The ``ZB_ZDO_SIGNAL_LEAVE`` signal is generated twice during Zigbee Coordinator factory reset.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-9714: Device association fails if the Request Key packet is retransmitted
   If the Request Key packet for the TCLK is retransmitted and the coordinator sends two new keys that are different, a joiner logic error happens that leads to unsuccessful key verification.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (maciejpietras) [inline doc/nrf/known_issues.rst:352]: KRKNWK-9743 be removed from NCS1.7.0

File: doc/nrf/known_issues.rst [hunk 5]
```diff
@@ -338,22 +339,22 @@ KRKNWK-6408: ``diag`` command not supported
...
 
 KRKNWK-9743 Timer cannot be stopped in Zigbee routers and coordinators
   The call to the ``zb_timer_enable_stop()`` API has no effect on the timer logic in Zigbee routers and coordinators.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (maciejpietras) [inline doc/nrf/known_issues.rst:357]: KRKNWK-10490 shall be removed from NCS.1.7.0

File: doc/nrf/known_issues.rst [hunk 6]
```diff
@@ -440,24 +441,24 @@ KRKNWK-7831: Factory reset broken on coordinator with Zigbee shell
...
   After the reset of OTA Upgrade Client, the client will start the OTA upgrade process from the beginning instead of continuing the previous process.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 KRKNWK-6318: Device assert after multiple Leave requests
   If a device that rejoins the network receives Leave requests several times in a row, the device could assert.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```
Comment (maciejpietras) [inline doc/nrf/known_issues.rst:454]: KRKNWK-6071 shall be removed from NCS1.7.0

File: doc/nrf/known_issues.rst [hunk 7]
```diff
@@ -440,24 +441,24 @@ KRKNWK-7831: Factory reset broken on coordinator with Zigbee shell
...
 
 KRKNWK-6071: ZBOSS alarms inaccurate
   On average, ZBOSS alarms last longer by 6.4 percent than Zephyr alarms.
 
   **Workaround:** Use Zephyr alarms.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```
Comment (maciejpietras) [inline doc/nrf/known_issues.rst:461]: KRKNWK-5535 shall be removed from NCS1.7.0

**Review by maciejpietras — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst [hunk 8]
```diff
@@ -569,7 +570,7 @@ Subsystems
 Bluetooth LE
 ============
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
 
 NCSDK-9820: The :ref:`peripheral_lbs` - If **Button 1** is pushed and released while holding one of the other buttons, the notification for release is the same as for press.
```
Comment (grochu) [inline doc/nrf/known_issues.rst:575]: NCSDK-9820 is fixed for NCS v. 1.7.0

**Review by grochu — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst [hunk 9]
```diff
@@ -1071,15 +1072,15 @@ KRKNWK-8842: MPSL does not support nRF21540 revision 1 or older
 802.15.4 Radio driver
 =====================
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-2 v1-4-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-2 v1-4-0
```
Comment (czeslawmakarski) [inline doc/nrf/known_issues.rst:1075]: This was fixed in NCS 1.7.0
```suggestion
.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-2 v1-4-0
```

**Review by czeslawmakarski — CHANGES_REQUESTED** [review]
One Known Issue was fixed in NCS 1.7.0

**Comment by greg-fer** [comment]
@grochu , @rugeGerritsen , @maciejpietras , @czeslawmakarski , updated, please check again.

**Review by rugeGerritsen — APPROVED** [review]

**Review by czeslawmakarski — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 10]
```diff
@@ -244,19 +245,19 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (MarekPorwisz) [inline doc/nrf/known_issues.rst:248]: fixed in 1.7

File: doc/nrf/known_issues.rst [hunk 11]
```diff
@@ -244,19 +245,19 @@ Missing :file:`CMakeLists.txt`
...
 
 KRKNWK-10633: Incorrect data when using ACK-based Probing with Link Metrics
   When using the ACK-based Probing enhanced with Link Metrics, the Thread Information Element contains fixed data instead of the correct Link Metrics data for the acknowledged frame.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (MarekPorwisz) [inline doc/nrf/known_issues.rst:253]: still in progress, but should be fixed for 1.7

File: doc/nrf/known_issues.rst [hunk 12]
```diff
@@ -244,19 +245,19 @@ Missing :file:`CMakeLists.txt`
...
 
 KRKNWK-10467: Security issues for retransmitted frames with Thread 1.2
   The Thread 1.2 current implementation does not guarantee that all retransmitted frames will be secured when using the transmission security capabilities of the radio driver.
   For this reason, OpenThread retransmissions are disabled by default when the :kconfig:`CONFIG_NRF_802154_ENCRYPTION` Kconfig option is enabled.
   You can enable the retransmissions at your own risk.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-11037:  ``Udp::GetEphemeralPort`` can cause infinite loop
```
Comment (MarekPorwisz) [inline doc/nrf/known_issues.rst:262]: fixed in 1.7

File: doc/nrf/known_issues.rst [hunk 13]
```diff
@@ -849,7 +850,7 @@ Flash commands only program one core
 
   **Workaround:** Execute the flash command from inside the build directory of the child image that is placed on the other core (for example, :file:`build/hci_rpmsg`).
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:853]: @mbolivar-nordic  or @tejlmand  this is fixed now right? At least my west is telling me that it is splitting the hex file and programming both.

Comment (hakonfam) [inline doc/nrf/known_issues.rst:853]:
```suggestion
.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
```

Comment (hakonfam) [reply doc/nrf/known_issues.rst:853]: Right @tejlmand  or @mbolivar-nordic ?

**Review by hakonfam — CHANGES_REQUESTED** [review]

**Review by grochu — APPROVED** [review]

**Review by maciejpietras — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 13 above)
Comment (mbolivar-nordic) [reply doc/nrf/known_issues.rst:861]: > Right @tejlmand or @mbolivar-nordic ?

This should be fixed as far as I know, yes

File: doc/nrf/known_issues.rst (hunk 11 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst:253]: @MarekPorwisz , any updates? Is this fixed now?

Comment (MarekPorwisz) [reply doc/nrf/known_issues.rst:253]: Yes, it is fixed

**Comment by canisLupus1313** [comment]
Please add known issue stating that new project won't be created correctly in VS code if it's based on openthread sample.

**Comment by greg-fer** [comment]
> Please add known issue stating that new project won't be created correctly in VS code if it's based on openthread sample.

This requires further discussion with @Avalei and @wbober , who are coordinating the VS Code extension documentation release.
I would anyway also need the JIRA and workaround for this issue.

File: doc/nrf/known_issues.rst [hunk 14]
```diff
@@ -865,7 +875,7 @@ NCSDK-7982: partition manager: Incorrect partition size linkage from name confli
 DFU and FOTA
 ============
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:878]: This seems to have been fixed a while ago (oct 2020)

https://github.com/nrfconnect/sdk-nrf/pull/3197

**Review by hakonfam — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst [hunk 15]
```diff
@@ -61,15 +62,15 @@ nRF9160
 Asset tracker
 =============
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:65]:
```suggestion
.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

File: doc/nrf/known_issues.rst [hunk 16]
```diff
@@ -179,14 +180,14 @@ Modem FW reset on debugger connection through SWD
   If a debugger (for example, J-Link) is connected via SWD to the nRF9160, the modem firmware will reset.
   Therefore, the LTE modem cannot be operational during debug sessions.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:183]: Adding 1.7.0 here is correct.

File: doc/nrf/known_issues.rst [hunk 17]
```diff
@@ -209,12 +210,12 @@ NCSDK-9786: Wrong FLASH_PAGE_ERASE_MAX_TIME_US for the nRF53 network core
 
   **Workaround:** Increase ``FLASH_PAGE_ERASE_MAX_TIME_US`` (defined in :file:`ncs/zephyr/soc/arm/nordic_nrf/nrf53/soc.h`) from 44850UL to 89700UL (the same value as for the application core).
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:213]:
```suggestion
.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

This is fixed

File: doc/nrf/known_issues.rst [hunk 18]
```diff
@@ -555,7 +564,7 @@ NCSDK-8304: HID configurator issues for peripherals connected over Bluetooth® L
 Pelion
 ======
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:567]: @emob-nordic the ticket is closed with a linked issue, should this be removed?

File: doc/nrf/known_issues.rst [hunk 19]
```diff
@@ -835,7 +845,7 @@ KRKNWK-7827: Application build system is not aware of the settings partition
     Set :kconfig:`CONFIG_USE_DT_CODE_PARTITION` Kconfig option to ``y``.
     Make sure that the code partition is defined and chosen correctly (``offset`` and ``size``).
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:848]: Adding 1.7.0 here is correct.

**Review by hakonfam — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst [hunk 20]
```diff
@@ -1474,7 +1498,7 @@ NCSDK-9992: Multiple extra CMake options applied as single option.
 Toolchain
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:1498]: @tejlmand , could you check this section and the SES section above? Thanks!

**Review by joerchan — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 18 above)
Comment (emob-nordic) [reply doc/nrf/known_issues.rst:567]: It should be removed, agreed. @greg-fer please do not add `v1-7-0` tag here.

**Review by MarekPieta — APPROVED** [review]
nRF Desktop issue may still be replicated.

**Review by maciejpietras — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 21]
```diff
@@ -1463,7 +1487,7 @@ NCSDK-8372: Project name collision causes SES Nordic Edition to load the wrong p
   **Workaround:** If the path in :guilabel:`Build Directory` points to the wrong project, select the correct project by using the :guilabel:`...` button for :guilabel:`Projects` and navigating to the correct project location.
   The build directory will update automatically.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
 
 NCSDK-9992: Multiple extra CMake options applied as single option.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:1492]: Task still in progress...

**Review by MarekPorwisz — APPROVED** [review]

**Review by hakonfam — APPROVED** [review]

**Review by emob-nordic — APPROVED** [review]

**Review by nordic-hani — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Comment by simensrostad** [comment]
@greg-fer
Known issue with Asset Tracker v2 configured for Azure IoT Hub: CIA-351
 - If a Device Firmware Upgrade (FOTA) is scheduled when the device is in PSM, there is the likelihood of the firmware image being downloaded twice.

**Review by mbolivar-nordic — APPROVED** [review]

**Review by simensrostad — APPROVED** [review]

**Review by totyz — CHANGES_REQUESTED** [review]
Zigbee

.. rst-class:: v1-7-0

KRKNWK-11330: NCP does not work on nRF5340
  NCP HOST is not able to setup communication with NCP SOC on nRF5340dk_nrf5340_cpuapp

**Comment by greg-fer** [comment]
@tejlmand confirmed the issues for Toolchain and SES are still valid for 1.7.0

File: doc/nrf/known_issues.rst [hunk 22]
```diff
@@ -19,9 +19,10 @@ Use the drop-down filter to see known issues for previous releases and check if
 
    <select name="versions" id="versions-select">
      <option value="all">All versions</option>
-	 <option value="v1-6-1" selected>v1.6.1</option>
-	 <option value="v1-6-0">v1.6.0</option>
-	 <option value="v1-5-1">v1.5.1</option>
+     <option value="v1-7-0" selected>v1.7.0</option>
+     <option value="v1-6-1">v1.6.1</option>
+	   <option value="v1-6-0">v1.6.0</option>
+	   <option value="v1-5-1">v1.5.1</option>
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:25]:
```suggestion
     <option value="v1-6-1">v1.6.1</option>
     <option value="v1-6-0">v1.6.0</option>
     <option value="v1-5-1">v1.5.1</option>
```

**Review by umapraseeda — APPROVED** [review]
LGTM

**Review by hakonfam — CHANGES_REQUESTED** [review]
image

https://github.com/nrfconnect/sdk-nrf/pull/5475/files#diff-a735de3a5c2dadf8c6e9cb659c0a3b14deb436c036827a2880665b9059d6f5f4R1499

This issue has been re-introduced (a downstream patch was lost). Should we add the `1-7-0` class here, and re-use the JIRA ticket, or should we create a new entry ?

File: doc/nrf/known_issues.rst [hunk 23]
```diff
@@ -943,22 +976,25 @@ Unstable NFC tag samples
...
+
+NCSDK-8232: Secure Partition Manager and application building together
+  It is not possible to build and program :ref:`secure_partition_manager` and the application individually.
+
+.. rst-class:: v1-5-1 v1-5-0
+
+CIA-248: Samples with default SPM config fails to build for ``thingy91_nrf9160_ns``
+   All samples using the default SPM config fails to build for the ``thingy91_nrf9160_ns``  build target if the sample is not set up with MCUboot.
+
+   **Workaround:** Use the master branch.
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:997]:
```suggestion
   **Workaround:** Use the master branch.
   
   .. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0

NCSDK-5156: The size of the non-secure callable area is wrong when the number of secure services exceeds 63
  If the number of services is between 64 and 255, the number is too large, resulting in up to 4096 of wasted space.
  If the number of services exceeds 255, the HW configuration becomes invalid.
  The default number of services is much smaller than 64.

  **Workaround:** Keep number of services below 256.
```

**Review by hakonfam — CHANGES_REQUESTED** [review]
We need to add an entry here. Let me know if I should create a PR against this branch instead of this proposal.

**Comment by greg-fer** [comment]
> This issue has been re-introduced (a downstream patch was lost). Should we add the 1-7-0 class here, and re-use the JIRA ticket, or should we create a new entry ?

@hakonfam , I think it's ok to re-use (re-open) the existing JIRA. Creating a new entry could be confusing for customers. Main question is: is the issue 100% identical now to the one closed before 1.5.0?

**Comment by hakonfam** [comment]
> @hakonfam , I think it's ok to re-use (re-open) the existing JIRA. Creating a new entry could be confusing for customers. Main question is: is the issue 100% identical now to the one closed before 1.5.0?

Yes, the issue is identical.

File: doc/nrf/known_issues.rst [hunk 24]
```diff
@@ -943,22 +976,34 @@ Unstable NFC tag samples
...
 NCSIDB-114: Default logging causes crash
   Enabling default logging in the :ref:`secure_partition_manager` sample makes it crash if the sample logs any data after the application has booted (for example, during a SecureFault, or in a secure service).
   At that point, RTC1 and UARTE0 are non-secure.
 
   **Workaround:** Do not enable logging and add a breakpoint in the fault handling, or try a different logging backend.
 
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+NCSDK-5156: Size of the non-secure callable area is wrong when the number of secure services exceeds 63
+  If the number of services is between 64 and 255, the number is too large, resulting in up to 4096 B of wasted space.
```
Comment (hakonfam) [inline doc/nrf/known_issues.rst:990]:
```suggestion
  If the number of services is between 64 and 255, the number is too large, resulting in up to 4096 bytes of wasted space.
```

Or is the capital B intentional?

**Review by hakonfam — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 24 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst:990]: Yes, B is OK, we don't need to spell out the unit in text.

**Comment by MarekPieta** [comment]
@greg-fer, there is one more for nRF Desktop:
```
.. rst-class:: v1-7-0

NCSDK-11378: An empty HID boot report is not forwarded to host computer by nRF Desktop dongle on peripheral disconnection. The host computer may not receive information that key that was previously reported as pressed was released.

Workaround: Do not enable HID boot protocol on nRF Desktop dongle.
```

**Comment by jhirsi** [comment]
@greg-fer, please, check following also, thanks. 

There is also one quite severe known issue for samples/nrf9160/modem_shell that should be mentioned, fix available but didn't get up merged:

`.. modem_shell:: v1-7-0

NCSDK-11033: Dial up usage with MoSh PPP does not work and causes DK crashing when connected with PC.

Workaround: Fix available in Zephyr. Pick the fix manually for a Zephyr bug: https://github.com/zephyrproject-rtos/zephyr/issues/38516`

**Comment by greg-fer** [comment]
> @greg-fer, please, check following also, thanks.

@jhirsi , the issue you mentioned is now listed in nRF9160/Other issues. Please let me know about the NCSDK-11033 issue later today.

File: doc/nrf/known_issues.rst [hunk 25]
```diff
@@ -1090,7 +1150,7 @@ KRKNWK-6255: RSSI parameter adjustment is not applied
 
     **Workaround:** To apply RSSI parameter adjustments, cherry-pick the commits in `hal_nordic PR #88 <https://github.com/zephyrproject-rtos/hal_nordic/pull/88>`_, `sdk-nrfxlib PR #381 <https://github.com/nrfconnect/sdk-nrfxlib/pull/381>`_, and `sdk-zephyr PR #430 <https://github.com/nrfconnect/sdk-zephyr/pull/430>`_.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
```
Comment (jciupis) [inline doc/nrf/known_issues.rst:1154]:
```suggestion

.. rst-class:: v1-7-0

KRKNWK-11384: At the rare occasions during the use of BLE and 802.15.4 multiprotocol the device might assert.

KRKNWK-11204: Transmitting the 802.15.4 frame with the improperly populated Auxilliary Security Header might result in assert.
```

**Review by jciupis — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst (hunk 25 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst:1158]: 11384 is added.
11204 is closed, which poses a question if you're planning to update it. If not, it is a limitation and should go into the changelog in sdk-nrfxlib. If yes, it requires a new JIRA.

Comment (greg-fer) [reply doc/nrf/known_issues.rst:1158]: Checked with @czeslawmakarski . The second issue is a limitation and will be added to https://github.com/nrfconnect/sdk-nrfxlib/pull/549

**Review by tomchy — APPROVED** [review]

**Review by jciupis — APPROVED** [review]

**Comment by de-nordic** [comment]
Known issue to mcumgr:
```
   JIRA-11432 Erasing secondary slot returns error response
        Trying to erase secondary slot results in an error response. Slot is still erased.
        
        Issue is only visible when application is compiled for multi-image.
```

**Review by de-nordic — APPROVED** [review]
mcumgr, DFU: OK.

**Review by b-gent — APPROVED** [review]

**Pull request merged**